README.md

```diff
@@ -1,2 +1,56 @@
 # sentry.js
 Helper to connect Sentry to tRPC &amp; Express.
+
+
+## Install
+
+```sh
+npm install @altipla/sentry
+```
+
+
+## Usage
+
+### Express
+
+```ts
+import express from 'express'
+import { expressRequestHandler, expressErrorHandler } from '@altipla/sentry'
+
+let app = express()
+app.use(expressRequestHandler())
+
+app.get('/', (req, res) => {
+  res.send('Hello World!')
+})
+//(...add more routes here...)
+
+app.use(expressErrorHandler())
+```
+
+
+### tRPC
+
```

Comment:
Vamos a explicar qué es lo que estamos aportando en el ejemplo, que no lo van a adivinar.

Set onError when connecting the tRPC routers. For example with the Express connector: